btax/parameters.py

```diff
@@ -68,6 +68,18 @@ def translate_param_names():
 	# btax_depr_allyr_gds_Switch
 	# btax_depr_allyr_tax_Switch
 
+	class_list = [3, 5, 7, 10, 15, 20, 25, 27.5, 39]
+
+	user_bonus_deprec={'3':btax_depr_3yr_exp, '5':btax_depr_5yr_exp,
+					'7':btax_depr_7yr_exp, '10':btax_depr_10yr_exp,
+					'15':btax_depr_15yr_exp, '20':btax_depr_20yr_exp,
+					'25':btax_depr_25yr_exp, '27.5':btax_depr_27_5yr_exp,
+					'39':btax_depr_39yr_exp}
+	user_deprec_system={'3':btax_depr_3yr_exp, '5':btax_depr_5yr_exp,
```

Comment:
One thing I note here @jdebacker  is that `user_bonus_deprec` and `user_deprec_system` become dictionaries by asset year class(with float values), but in the existing code `bonus_deprec` is a scalar float and `deprec_system` is not referenced.  Does the code need to be run for each key value in the `user_bonus_deprec` dictionary?